airtime_mvc/application/models/StreamSetting.php

```diff
@@ -464,7 +464,7 @@ public static function setDjLiveStreamPort($value)
 
     public static function getDjLiveStreamPort()
     {
-        return self::getValue("dj_live_stream_port", 8001);
+        return self::getValue("dj_live_stream_port", 8002);
```

Comment:
Sounds ok, I was assuming it would only use different mountpoints. We should NAT 8002 in vagrant per default as well in this case.